examples/ding-dong-bot-oop.py

```diff
@@ -60,13 +60,15 @@ async def on_message(self, msg: Message):
 
             hd_file_box: FileBox = await image.hd()
             await hd_file_box.to_file('./hd-image.jpg', overwrite=True)
-
-            thumbnail_file_box: FileBox = await image.thumbnail()
-            await thumbnail_file_box.to_file('./thumbnail-image.jpg', overwrite=True)
-            artwork_file_box: FileBox = await image.artwork()
-            await artwork_file_box.to_file('./artwork-image.jpg', overwrite=True)
```

Comment:
I think those methods to transfer `Image` to `Filebox` are the same, one of them will work, rather than all run aside. In fact, I can note him off rather than delete.

Regarding this error, I would be happy to fix it if I have time, so which part do you think is the problem? I can try to fix it.